### DISCUSSION TITLE
How to generate test031_many_gwf

### DISCUSSION THREAD

**Discussion by lj-cug** [modflowpy/flopy#1756] [General]
If I use modeflow6-parallel developed by Verkaik, which support coarse-grained parallelization (MPI) for many GWFs exchange case.
So it's useful for me to generate the necessary input files like the test models-- test031_many_gwf, in which 9 domains were generated for GWF exchanges.
Can I generate the many_gwf input file using Flopy?

**Comment by christianlangevin** [comment]
We are currently working on a model splitter for flopy that will help do what you need.  The splitter will take a single model and split it into multiple models and exchanges.  A primary purpose for the splitter will be to support the parallel MODFLOW capabilities that are on the way.  Until the splitter is available, which should be within the next couple of months, you'll need to create the datasets manually, perhaps with some simple python code.  You are a little early on all this.  Much more support is on the way.

**Reply by lj-cug** [reply]
Thank you very much for you reply and continuous development of Flopy. The splitter will be very useful for setting up parallel modflow6. At present I will manually config the parallel modflow6.

**Reply by wpbonelli** [reply]
The model splitter is now in the stable flopy release
